### PR TITLE
ci: Update renovate schedule and enable automerge

### LINKED ISSUE #227: Fine tune the renovate config
* Once two weeks still produce too many PRs, maybe once every 4 weeks
 * Add rule for certain packages to be checked even less frequent
 * Automerge some of the PRs
 * Automate the sandbox upgrade to accomodate the automerge (cloud scheduler calls a webhook to trigger reload workflow with force flag, the daemon will restart. Since the imagePullPolicy is `Always`, a new nightly build will be used)

We should also come up with a release schedule to match this.

### PR DESCRIPTION
**Pull request by cynful**
Closes #227

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-  "schedule": ["on Monday every 2 weeks"],
+  "schedule": ["before 8am on the first day of the month"],
   "packageRules": [
     {
-      "depTypeList": [
-        "require"
-      ],
+      "depTypeList": ["require"],
+      "updateType": ["minor", "patch"],
+      "automerge": true,
```
Comment (wyardley) [inline renovate.json:10]: I'm guessing we don't want to enable automerge for this project
Plus, we'd likely run into codeowners issues anyway.

File: renovate.json [hunk 2]
```diff
@@ -2,12 +2,12 @@
   "extends": [
     "config:base"
   ],
-  "schedule": ["on Monday every 2 weeks"],
+  "schedule": ["before 8am on the first day of the month"],
```
Comment (wyardley) [inline renovate.json:5]: Maybe we should do a less predictable day?

File: renovate.json (hunk 1 above)
Comment (cynful) [reply renovate.json:10]: I thought we wanted to turn on automerge, as mentioned in #227 ? 🤔

Comment (wyardley) [reply renovate.json:10]: We'd need two separate configs then, one for automerge qualifying stuff and one for now. But, absent some other kind of bot like bulldozer, I don't think we can do automerge with codeowners / branch protection enabled.
So let's get this working without any automerge enabled at the renovate level. if we want to create some labels that would let us use another tool to automerge, that could make some sense

Let's avoid using policy-bot for this use case if we can.

File: renovate.json (hunk 2 above)
Comment (cynful) [reply renovate.json]: scheduling for 3rd day of the month

**Pull request merged**